.pre-commit-config.yaml

```diff
@@ -29,6 +29,7 @@ repos:
           - '--args=--only=terraform_naming_convention'
           - '--args=--only=terraform_required_version'
           - '--args=--only=terraform_required_providers'
+          - '--args=--only=terraform_unused_required_providers'
```

Comment:
nifty little lint check to make sure we remove providers when they aren't being utilized